packages/server/postgres/migrations/1719351990570_TimelineEvent-phase2.ts

```diff
@@ -0,0 +1,103 @@
+import {Kysely, PostgresDialect} from 'kysely'
```

Comment:
This file should not be here, isn't it? 🤔 I wonder if there are other files in the PR that shouldn't be here too 🤔